.travis.yml

```diff
@@ -34,9 +34,10 @@ install:
   - $PYTHON_EXE -m easy_install --version
   - $PYTHON_EXE -m pip --version
   - $PYTHON_EXE -m tox --version
-  - $PYTHON_EXE -m pip install -U setuptools
```

Comment:
You are right!
We don't need run the tests 2 times.
I changed for just test the installation of the package without errors.